org.mozilla.Firefox.desktop

```diff
@@ -1,34 +1,271 @@
 [Desktop Entry]
-Type=Application
+Version=1.0
 Name=Firefox
-Name[ar]=فايرفوكس
-Name[bn]=ফায়ার ফক্স
-Name[es]=Firefox
-Name[fr]=Firefox
-Name[hi]=Firefox
-Name[id]=Firefox
-Name[pt_BR]=Firefox
-Name[ro]=Firefox
-Name[th]=Firefox
-Name[vi]=Firefox
-Name[zh_CN]=Firefox
-Name[zh_TW]=Firefox
-GenericName=Web browser
-Comment=Mozilla's popular web browser
-Comment[ar]=متصفح ويب موزيلا الشهير
-Comment[es]=El navegador popular de Mozilla
-Comment[fr]=Fameux navigateur web de Mozilla
-Comment[hi]=Mozilla का लोकप्रिय वेब ब्राउज़र
-Comment[id]=Peramban web populer dari Mozilla
-Comment[pt_BR]=O famoso navegador da Mozilla
-Comment[ro]=Browser-ul web popular Mozilla
-Comment[th]=เว็บเบราว์เซอร์ยอดนิยมของ Mozilla
-Comment[vi]=Trình duyệt web phổ biến của Mozilla
-Comment[zh_CN]=Mozilla 出品的一款非常受欢迎的 Web 浏览器
-Comment[zh_TW]=Mozilla 的熱門網頁瀏覽器
+GenericName=Web Browser
+GenericName[ca]=Navegador web
+GenericName[cs]=Webový prohlížeč
+GenericName[es]=Navegador web
+GenericName[fa]=مرورگر اینترنتی
+GenericName[fi]=WWW-selain
+GenericName[fr]=Navigateur Web
+GenericName[hu]=Webböngésző
+GenericName[it]=Browser Web
+GenericName[ja]=ウェブ・ブラウザ
+GenericName[ko]=웹 브라우저
+GenericName[nb]=Nettleser
+GenericName[nl]=Webbrowser
+GenericName[nn]=Nettlesar
+GenericName[no]=Nettleser
+GenericName[pl]=Przeglądarka WWW
+GenericName[pt]=Navegador Web
+GenericName[pt_BR]=Navegador Web
+GenericName[sk]=Internetový prehliadač
+GenericName[sv]=Webbläsare
+Comment=Browse the Web
+Comment[ca]=Navegueu per el web
+Comment[cs]=Prohlížení stránek World Wide Webu
+Comment[de]=Im Internet surfen
+Comment[es]=Navegue por la web
+Comment[fa]=صفحات شبکه جهانی اینترنت را مرور نمایید
+Comment[fi]=Selaa Internetin WWW-sivuja
+Comment[fr]=Navigue sur Internet
+Comment[hu]=A világháló böngészése
+Comment[it]=Esplora il web
+Comment[ja]=ウェブを閲覧します
+Comment[ko]=웹을 돌아 다닙니다
+Comment[nb]=Surf på nettet
+Comment[nl]=Verken het internet
+Comment[nn]=Surf på nettet
+Comment[no]=Surf på nettet
+Comment[pl]=Przeglądanie stron WWW
+Comment[pt]=Navegue na Internet
+Comment[pt_BR]=Navegue na Internet
+Comment[sk]=Prehliadanie internetu
+Comment[sv]=Surfa på webben
 Exec=firefox %u
 Icon=org.mozilla.Firefox
 Terminal=false
+Type=Application
+MimeType=text/html;text/xml;application/xhtml+xml;application/vnd.mozilla.xul+xml;text/mml;x-scheme-handler/http;x-scheme-handler/https;-X-Endless-LaunchMaximized=true;
```

Comment:
`X-Endless-LaunchMaximized` should go to its own line and without a `;` at the end.